roles/helm-ansible-base/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+# If true, apply the changes by either creating or deleting resources in
+# kubectl. Set this to false if you want an opportunity to modify the resource
+# file before it gets applied. If false, you will be responsible for using
+# the manifest with kubectl or similar.
+hb_apply: true
```

Comment:
My initial thought when I saw this was `kubectl apply ...`, I see we have a separate `hb_k8s_action` but it was enough of a hiccup to give me pause. Could a user not just set `hb_k8s_action` to false or empty string and we skip the `kubectl create|delete`?

Reply:
Interesting idea. The role does different things depending on whether the action is "provision" vs "deprovision", so it still needs to know that regardless of whether any changes get "applied" at the end. If apb-base passed the action (provision/deprovision) as an ansible extra-var, we could use that instead of `hb_k8s_action`.

So I think we still need both of these for now.